src/source/PeerConnection/PeerConnection.c

```diff
@@ -923,15 +923,16 @@ STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection pRtcPeerConne
 
 /// @brief parses string of form "$number $whatever" returns $number as uint32
 /// @return 0 if value is not parsable or null
-static UINT32 parseExtId(PCHAR extmapValue)
+UINT32 parseExtId(PCHAR extmapValue)
 {
     UINT32 extid = 0;
-    if (extmapValue != NULL && STRCHR(extmapValue, ' ') != NULL) {
-        if (STATUS_SUCCEEDED(STRTOUI32(extmapValue, STRCHR(extmapValue, ' '), 10, &extid))) {
-            return extid;
-        }
+    if (extmapValue == NULL && STRCHR(extmapValue, ' ') == NULL) {
```

Comment:
Another nit:

I would do something like this

UINT32 extid = 0;
    if (extmapValue != NULL && STRCHR(extmapValue, SPACE_CHAR && STATUS_SUCCEEDED(STRTOUI32(extmapValue, STRCHR(extmapValue, ' '), 10, &extid)) != NULL) {
    return extid;
} else {
return 0;
}

Feel free to ignore this BTW

Reply:
i tried it, clang puts the statement on one line 
looks super ugly to me :man_shrugging: